config/crd/bases/storage.dell.com_containerstoragemodules.yaml

```diff
@@ -2639,10 +2667,6 @@ spec:
                     name:
                       description: Name is name of ContainerStorageModule modules
                       type: string
-                    openshift:
```

Comment:
this should also be removed from deploy/crds/storage.dell.com.crds.all.yaml
`make manifests && make generate && make install`

Reply:
Thank you, that part was taken care with `make install`